### PR TITLE
4 add together ai as llm

### LINKED ISSUE #4: add-together-ai-as-llm

### PR DESCRIPTION
**Pull request by AndrewKPascual**

### CODE REVIEW

File: ingest/main.py
Comment (AGI-CEO) [inline ingest/main.py:1]: why edit this?

File: backend/app/utils/index.py [hunk 1]
```diff
@@ -32,14 +36,16 @@ def get_index():
         logger.info("Creating new index")
         # load the documents and create the index
         documents = SimpleDirectoryReader(DATA_DIR).load_data()
```
Comment (AGI-CEO) [inline backend/app/utils/index.py:38]: why use a simple directory reader?

**Review by AGI-CEO — CHANGES_REQUESTED** [review]
Address questions in comments

File: ingest/main.py
Comment (AndrewKPascual) [reply ingest/main.py:1]: It does not work for anyone unless "if __name__ == '__main__':  main()" is added

File: backend/app/utils/index.py (hunk 1 above)
Comment (AndrewKPascual) [reply backend/app/utils/index.py:38]: This was so people could test their prompt.

**Comment by AGI-CEO** [comment]
sloppy code.

**Pull request merged**